composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.4",
         "php-http/httplug": "^1.0.0-alpha2@dev",
```

Comment:
so instead of requiring promise, we should require at least alpha3. not sure why that was not picked in the first place.